<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proveedor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #263238;</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color:  white;
    border-style:  solid;
    border-width: 1.5px;
    border-color: #0097a7;
	border-radius: 10px;
	padding: 10px;
}

QComboBox {
    color:  white;
    border-style:  solid;
    border-width: 1.5px;
    border-color: #0097a7;
	border-radius: 10px;
	padding: 10px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,8">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>Crear Proveedor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLineEdit" name="input_razon_social">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    color:  white;
    border-style:  solid;
    border-width: 1.5px;
    border-color: #0097a7;
	border-radius: 10px;
	padding: 10px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Razón Social</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_ruc">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    color:  white;
    border-style:  solid;
    border-width: 1.5px;
    border-color: #0097a7;
	border-radius: 10px;
	padding: 10px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Ruc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_direccion">
           <property name="placeholderText">
            <string>Dirección</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_telefono">
           <property name="placeholderText">
            <string>Teléfono</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_representante">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Representante</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_save">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color:  white;
	background-color: #0097a7;
	border-radius: 10px;
	padding: 10px;
}</string>
           </property>
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
